68 create python file that collects static data

PedroPianna opened the pull request:
1. Refactored code for better readability. Also documented some functions
2. [data_exporter.py](https://github.com/Ninjagoat73/project2-1_group15/blob/develop/data-collection/data_exporter.py) now adds entry to a static.csv file, instead of creating a new CSV file for each run
3. run_id has been delegated to issue #70, so instead of generating the run_id, the function takes one as parameter

#### Closes #68 

### Types of change(s)
- [ ] Bug fix
- [x] New feature
- [x] Code refactor
- [ ] Breaking change
- [x] Documentation update
- [ ] Other (please describe)

PedroPianna committed:
refactor data_exporter.py and append to single static.csv file instead of creating a new one every time

also changed the run_id generation, since it will be handled bu the 'main' script

PedroPianna committed:
further refactoring on data_exporter.py

PedroPianna committed:
added documentation and function now returns the status (success or not)

zetenyLevay commented:
looks great, do you want the training.csv and summarize.csv be done in different files (single responsibility ig)?

PedroPianna commented:
> looks great, do you want the training.csv and summarize.csv be done in different files (single responsibility ig)?

Exactly.  I've moved the code for those to a separate file. Haven't pushed it tho. I could do it, or just keep it on my laptop locally until we have to use it. Also I guess @Predrag-Tomic is doing the training.csv, as he is collecting data from the results folder? What do you think?

zetenyLevay commented:
Thats good, you should push the files tho, so if someone makes a branch of it they know to use those, what do you think?, I will work on the hardware info part in stats recorder, shouldnt take too long.

PedroPianna committed:
template for functions that will create the summary.csv and training.csv files

so far the functions only have the headers for the respective csv files, but those can be useful in the future

PedroPianna commented:
> Thats good, you should push the files tho, so if someone makes a branch of it they know to use those, what do you think?, I will work on the hardware info part in stats recorder, shouldnt take too long.

Sure. Just did it. They basically have the headers for the csv files. That's good. Is there anything more we should do within this issue? Or do you think we can close it and merge?

zetenyLevay commented:
I think we can close and merge
